schema/data/portal/form_result.sql

```diff
@@ -31,4 +23,6 @@ select setval from
 setval('ggircs_portal.form_result_id_seq', (select max(id) from ggircs_portal.form_result), true)
 where setval = 0;
 
+select ggircs_portal.create_application_mutation_chain(2);
```

Comment:
Creating a draft application for facility 2. For test data, should be removed in production